war/src/main/less/form/layout.less

```diff
@@ -78,18 +78,23 @@
 
 .jenkins-help-button {
   position: relative;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
   width: 20px;
   height: 20px;
   min-width: 20px;
   min-height: 20px;
   margin-left: 1ch;
-  display: inline-flex;
-  justify-content: center;
-  align-items: center;
   line-height: 20px;
-  color: var(--text-color)!important;
+  color: var(--text-color) !important;
   border-radius: 100%;
 
+  svg {
+    width: 50%;
+    height: 50%;
+  }
+
   &::before {
     content: "?";
```

Comment:
```suggestion
    content: "";
```
I took care of that in a previous PR, yet it sneaked its way in again.
![](https://i.imgur.com/1k15y2o.png)
turns into
![](https://i.imgur.com/WqAwRMT.png)
The theme plugins are unaffected by this change, it solely takes care of browser extensions like dark reader.

Reply:
Ah blimey, no idea how that got back in there. I've removed it now and reworked slightly how the button works so it's consistent across browsers/OSs. Cheers 👍